actionpack/test/controller/integration_test.rb

```diff
@@ -315,8 +315,6 @@ def test_opens_new_session
     session1 = @test.open_session { |sess| }
     session2 = @test.open_session # implicit session
 
-    assert_respond_to session1, :assert_template, "open_session makes assert_template available"
-    assert_respond_to session2, :assert_template, "open_session makes assert_template available"
```

Comment:
Publish the plugin as a gem?

Date: Wed, 13 May 2015 07:14:31 -0700
From: notifications@github.com
To: rails@noreply.github.com
CC: tgx_world@hotmail.com
Subject: Re: [rails] Deprecate `assert_template`. (#20138)

In actionpack/test/controller/integration_test.rb:

> @@ -315,8 +315,6 @@ def test_opens_new_session
>      session1 = @test.open_session { |sess| }
>      session2 = @test.open_session # implicit session
>  
> -    assert_respond_to session1, :assert_template, "open_session makes assert_template available"
> -    assert_respond_to session2, :assert_template, "open_session makes assert_template available"

Would be nice to have both assert_template and assigns() available in a
plugin, so people who like that kind of thing can continue using it.
…
On Wed, May 13, 2015 at 12:38 PM, Guo Xiang Tan <notifications@github.com> wrote:
 In actionpack/test/controller/integration_test.rb
 <https://github.com/rails/rails/pull/20138#discussion_r30219276>:

 > @@ -315,8 +315,6 @@ def test_opens_new_session
 >      session1 = @test.open_session { |sess| }
 >      session2 = @test.open_session # implicit session
 >
 > -    assert_respond_to session1, :assert_template, "open_session makes assert_template available"
 > -    assert_respond_to session2, :assert_template, "open_session makes assert_template available"

 Duplicated in
 https://github.com/tgxworld/rails/blob/deprecated_assert_template/actionpack/test/dispatch/template_assertions_test.rb#L66

 ―
 Reply to this email directly or view it on GitHub
 <https://github.com/rails/rails/pull/20138/files#r30219276>.




―
Reply to this email directly or view it on GitHub.